src/components/TradeAssetSearch/components/CustomAssetAcknowledgement.tsx

```diff
@@ -74,14 +74,8 @@ export const CustomAssetAcknowledgement: React.FC<CustomAssetAcknowledgementProp
 
     // Add asset to the store
     dispatch(assetsSlice.actions.upsertAsset(asset))
-
-    // Add market data to the store
-    dispatch(
-      marketDataSlice.actions.setCryptoMarketData({
-        [asset.assetId]: { price: '0', marketCap: '0', volume: '0', changePercent24Hr: 0 },
-      }),
-    )
-
+    // Use the market API to get the market data for the custom asset
+    dispatch(marketApi.endpoints.findByAssetIds.initiate([asset.assetId]))
```

Comment:
This does nothing in this PR, but sets everything up so it'll work once the market provider for custom assets is added.